### PR TITLE
I thought I was being smart

### PR DESCRIPTION
**Pull request by TheBunnyMan123**

### CODE REVIEW

**Review by PoolloverNathan — DISMISSED** [review]
the antismart

File: common/build.gradle [hunk 1]
```diff
@@ -18,7 +18,7 @@ dependencies {
     compileOnly("com.neovisionaries:nv-websocket-client:$nv_websocket")
 
     // Figura common lib
-    modCompileOnly("org.figuramc:figura-common-intermediary:$figura_version+$minecraft_version")
+    modCompileOnly("org.figuramc:figura-common-intermediary:$figura_version+1.20.1")
```
Comment (PoolloverNathan) [inline common/build.gradle:21]: remember to revert this when 0.1.4+1.20.4 drops

**Review by PoolloverNathan — APPROVED** [review]
yeah do not let me code at 1am

**Comment by TheBunnyMan123** [comment]
> yeah do not let me code at 1am

@PoolloverNathan fapi version doesn't exist on 1.20.1

**Comment by TheBunnyMan123** [comment]
oh

**Comment by TheBunnyMan123** [comment]
what now

**Comment by TheBunnyMan123** [comment]
Nathan your flake

File: common/src/main/java/com/thekillerbunny/goofyplugin/mixin/FiguraAPIManagerMixin.java [hunk 2]
```diff
@@ -36,7 +36,7 @@
 import java.util.Set;
 import java.util.function.Function;
 
-@Mixin(value = FiguraAPIManager.class, remap = false)
+@Mixin(value = FiguraAPIManager.class)
```
Comment (PoolloverNathan) [inline common/src/main/java/com/thekillerbunny/goofyplugin/mixin/FiguraAPIManagerMixin.java:39]: bit hacky but works

File: flake.nix [hunk 3]
```diff
@@ -35,17 +35,33 @@
...
+          run1 = taskFor {
+            task = "runClient";
+            minecraft = "1.20.1";
+            fabric-api = "0.83.0";
+          };
+          run4 = taskFor {
+            task = "runClient";
+            minecraft = "1.20.4";
+            fabric-api = "0.97.0";
+          };
```
Comment (PoolloverNathan) [inline flake.nix:64]: Domain Expansion: Quintupled Code

File: gradle.properties [hunk 4]
```diff
@@ -21,7 +21,7 @@ authors = TheKillerBunny, PoolloverNathan
 # https://fabricmc.net/develop
 # https://modrinth.com/mod/fabric-api
 fabric_loader_version=0.14.21
-fabric_api_version=0.85.0
+fabric_api_version=0.97.0
```
Comment (PoolloverNathan) [inline gradle.properties:24]: I blame @PoolloverNathan

**Review by PoolloverNathan — COMMENTED** [review]
¯\\\_(ü)\_/¯

File: build.gradle [hunk 5]
```diff
@@ -33,10 +33,10 @@ subprojects {
     }
 
     dependencies {
-        minecraft "com.mojang:minecraft:$minecraft_version"
+        minecraft "com.mojang:minecraft:$minecraft_version_out"
 
         mappings loom.layered {
-			mappings("org.quiltmc:quilt-mappings:$minecraft_version+build.$mappings:intermediary-v2")
+			mappings("org.quiltmc:quilt-mappings:$minecraft_version_out+build.$mappings:intermediary-v2")
```
Comment (PoolloverNathan) [inline build.gradle:39]: mappings *probably* need to use the latest version

File: flake.nix [hunk 6]
```diff
@@ -35,17 +35,39 @@
...
+            fabric-api = "0.83.0";
+            loom = "1.2-SNAPSHOT";
+          };
+          run4 = taskFor {
+            task = "runClient";
+            minecraft = "1.20.1";
+            minecraft-out = "1.20.4";
+            fabric-api = "0.83.0";
+            loom = "1.2-SNAPSHOT";
+          };
```
Comment (PoolloverNathan) [inline flake.nix:70]: domain expansion: a fuckton more code

**Review by PoolloverNathan — APPROVED** [review]
YOLO

**Review by PoolloverNathan — APPROVED** [review]
Reapply "Revert "Revert "Reapply "fucking thanos snap the remap config"""

**Comment by PoolloverNathan** [comment]
IT BUILDS :partying_face:

**Comment by TheBunnyMan123** [comment]
nobody will ever understand how happy we are

**Comment by TheBunnyMan123** [comment]
making a release

**Pull request merged**